pkg/local_object_storage/blobstor/fstree/control.go

```diff
@@ -24,6 +24,11 @@ func (t *FSTree) Init() error {
 			t.writer = w
 		}
 	}
+
+	err = t.CleanUpTmp()
```

Comment:
I think it's too much. FSTree can be huge and it can delay the startup considerably.